src/PostgREST/Config.hs

```diff
@@ -101,6 +103,9 @@ instance Show LogLevel where
   show LogWarn  = "warn"
   show LogInfo  = "info"
 
+data OpenAPIMode = OAIFollowACL | OAIIgnoreACL | OAIDisabled
+  deriving Eq
```

Comment:
For "OpenAPI", I like OA better as an acronym than OAI, e.g. `OAFollowACL`, `OADisabled`. WDYT?

Reply:
Yeah, that's better. The OAI term from https://github.com/OAI, somehow stuck for me, but it's a different thing.